apps/dashboard/src/styles.css

```
@@ -1,8 +1,23 @@
 :root {
+  --color-background: #fafaf8;
+  --color-border: #e6e6e0;
+  --color-primary: #1a1a1a;
+  --color-panel: #ffffff;
+  --status-success: #3a8a55;
+  --status-warning: #b58a2b;
+  --status-danger: #b5413a;
+  --panel-radius: 8px;
   color-scheme: light;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)